runelite-client/src/main/java/net/runelite/client/plugins/skillcalculator/skills/AgilityAction.java

```diff
@@ -72,4 +75,17 @@ public boolean isMembers(final ItemManager itemManager)
 	{
 		return true;
 	}
+
+	@Override
+	public Set<AgilityBonus> getExcludedSkillBonuses()
+	{
+		final EnumSet<AgilityBonus> others = EnumSet.allOf(AgilityBonus.class);
+
+		if (this == WILDERNESS_AGILITY_COURSE_TICKET)
+		{
+			others.clear();
```

Comment:
This isn't future-proof, let's be more explicit about how we want it changed.
```suggestion
			others.remove(WILDERNESS_AGILITY_TICKET_11_TO_50);
			others.remove(WILDERNESS_AGILITY_TICKET_51_TO_100);
			others.remove(WILDERNESS_AGILITY_TICKET_100_PLUS);
```